roles/raspberrypi/tasks/main.yml

```
@@ -33,18 +33,29 @@
     - raspberry_pi|default(false)
     - ansible_facts.lsb.description|default("") is match("Debian.*buster")
 
+- name: Set detected_distribution to Raspbian (ARM64 on Debian Bullseye)
+  set_fact:
+    detected_distribution: Raspbian
+  when:
+    - ansible_facts.architecture is search("aarch64")
+    - raspberry_pi|default(false)
+    - ansible_facts.lsb.description|default("") is match("Debian.*bullseye")
```

I made the detection task a separate task for bullseye to be explicit for bullseye, but if prefered, this can be combined with the buster match.
A few comments on this would be appreciated.

Example change:
```suggestion
    - ansible_facts.lsb.description|default("") is match("Debian.*(buster|bullseye)")
```